docs/source/gettingstarted.rst

```diff
@@ -39,6 +39,11 @@ is to enable the NeuroDebian_ repository, and to install datalad as a system
 package::
 
   sudo apt-get install datalad
+  
+A current version of git-annex (as also provided by the NeuroDebian_ 
+repository) can be installed by typing::
+
+  sudo apt-get install git-annex
```

Comment:
At first, I used "pip install datalad" as recommended in the doc and consequently did not add the neurodebian repository before. 
Then, when installing git-annex ("sudo apt-get install git-annex") without having added the neurodebian repository I ran into an outdated version provided by the official debian repository. So I added the Neurodebian repository and did "sudo apt-get install git-annex-standalone" (and not just "sudo apt-get install git-annex"; my bad!)